tests/manual_test_etoro.py

```diff
@@ -0,0 +1,47 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2017-2018 reverendus
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import sys
+
+from pyexchange.etoro import EToroApi
+from pymaker.numeric import Wad
+
+etoro = EToroApi('https://7vriaeqd.hft.etorox.com', sys.argv[1], open('./.etoro-unencrypted-key', 'r').read(), 9.5)
+# etoro = EToroApi('https://7vriaeqd.hft.etorox.com', sys.argv[1], open(sys.argv[2], "r",encoding='utf-8').read(), 9.5)
```

Comment:
Parameterizing the URL and auth credentials would make it easier for others to test.

Reply:
Reverted back to `sys.argv` for host and credentials.